dandiapi/api/views/dashboard.py

```diff
@@ -82,6 +83,26 @@ def _users(self):
         )
 
 
+def mailchimp_csv_view(request: HttpRequest) -> HttpResponse:
+    """Generate a Mailchimp-compatible CSV file of all active users."""
+    # In production, there's a placeholder user with a blank email that we want
+    # to avoid.
+    users = User.objects.filter(is_active=True).exclude(email='')
+    data = users.values_list('email', 'first_name', 'last_name')
+
+    response = HttpResponse(
+        content_type='text/csv',
+        headers={
+            'Content-Disposition': 'attachment; filename="dandi_users_mailchimp.csv"',
+        },
+    )
+    writer = csv.writer(response)
```

Comment:
Now I'm being pedantic, but I find [csv.DictWriter](https://docs.python.org/3/library/csv.html#csv.DictWriter) to be harder to misuse.